src/wrapper.css

```diff
@@ -1,6 +1,8 @@
 .graphiql-explorer-root {
   font-size: var(--body-font-size) !important;
   font-family: var(--editor-font-family) !important;
+  overflow: auto !important;
+  height: calc(100vh - 60px);
```

Comment:
Wouldn't height of 100% work?

Reply:
No it would not - feel free to give it a try 

If you would use 100% this is would be the same as 100vh in this case. So the Top tabs (the 60px) would be too much with 100% and you would end up having two scrollbars 😌